checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
The test doesn't ensure that the annotation is stored in bytecode. It just tests whether the compiler crashes. I've renamed the file, to make all this clearer.